new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/TestActions.java

```diff
@@ -501,6 +538,12 @@ public static boolean deleteImage(String image) {
     return Docker.deleteImage(image);
   }
 
+  // ------------------------ Ingress -------------------------------------
+
+  public static List<String> getIngress(String namespace) throws Exception {
```

Comment:
change method name to listIngresses or getIngressList as you getting all the ingresses in the given namespace

Reply:
same applies to the method name in Traefik.java

Reply:
fixed